### PR TITLE
fix: createDataPool -> v2

### PR DESCRIPTION
**Pull request by joekendal**
Fixes the data pool resource by using the latest schema

### CODE REVIEW

File: propel/resource_data_pool.go [hunk 1]
```diff
@@ -64,6 +65,46 @@ func resourceDataPool() *schema.Resource {
...
+			// 	Description: "The tenant ID for restricting access between customers.",
+			// 	Elem: &schema.Resource{
+			// 		Schema: map[string]*schema.Schema{
+			// 			"columnName": {
+			// 				Type:     schema.TypeString,
+			// 				Required: true,
+			// 			},
+			// 		},
+			// 	},
+			// },
```
Comment (markandrus) [inline propel/resource_data_pool.go:107]: OK to un-comment this?

**Review by markandrus — COMMENTED** [review]
Looks good! Were you able to test it? One question about `tenant`

File: propel/resource_data_pool.go (hunk 1 above)
Comment (joekendal) [reply propel/resource_data_pool.go:107]: Haven't implemented that

**Comment by joekendal** [comment]
> Looks good! Were you able to test it? 

Yes, tested and its working correctly. I will update the docs and add tenant then I think it is good to go.

File: propel/resource_data_pool.go (hunk 1 above)
Comment (markandrus) [reply propel/resource_data_pool.go:107]: Ack, just tried myself and saw 👍

**Comment by senekis** [comment]
Cool @joekendal

**Comment by joekendal** [comment]
Also added an action to run the tests

**Review by senekis — APPROVED** [review]

**Pull request merged**